libshpool/src/daemon/server.rs

```diff
@@ -823,29 +831,35 @@ impl Server {
 
     /// Set up the environment for the shell, returning the right TERM value.
     #[instrument(skip_all)]
-    fn inject_env(
+    fn build_shell_env(
```

Comment:
(no action needed)
If I understand, we're rather creating the env vars we need and passing them around before injecting them (later), nice.